### PR TITLE
🚮 Remove Bento from various CI tasks, build systems, docs, and examples

### PR DESCRIPTION
**Pull request by danielrozenberg**
_Bento deprecation project_

TODO after merging:
- [ ] Delete the `bento_publishing` [environment](https://github.com/ampproject/amphtml/settings/environments) from this GitHub project

### CODE REVIEW

File: .github/workflows/publish-npm-packages.yml
Comment (estherkim) [inline .github/workflows/publish-npm-packages.yml:1]: undelete! we want to keep publishing amp packages

File: build-system/release-tagger/make-release.js
Comment (estherkim) [inline build-system/release-tagger/make-release.js:1]: remove this change from PR

File: build-system/test-configs/config.js [hunk 1]
```diff
@@ -98,15 +93,6 @@ const unitTestPaths = [
   'ads/**/test/unit/test-*.js',
   'extensions/**/test/*.js',
   'extensions/**/test/unit/*.js',
-  'src/bento/components/**/test/*.js',
-  'src/bento/components/**/test/unit/*.js',
-];
-
-const bentoUnitTestPaths = [
```
Comment (estherkim) [inline build-system/test-configs/config.js:105]: I see you're deleting the test-e2e files so shouldn't you delete these too?

Might be cleaner to delete all the test files in a separate PR

File: build-system/release-tagger/make-release.js
Comment (danielrozenberg) [reply build-system/release-tagger/make-release.js:1]: Done.

File: .github/workflows/publish-npm-packages.yml
Comment (estherkim) [reply .github/workflows/publish-npm-packages.yml:1]: oo you're right, I forgot we deprecated the amp component packages

File: build-system/test-configs/config.js (hunk 1 above)
Comment (estherkim) [reply build-system/test-configs/config.js:105]: Yeah nbd as long as you remember to delete the other bento test files.

**Review by erwinmombay — APPROVED** [review]

**Pull request merged**